### ISSUE TITLE
Movie sets not working

### ISSUE THREAD

**Issue by Ulrar** [jellyfin/jellyfin-kodi#615]
**Describe the bug**
Movie sets stopped working a while back, movies just appear all together as if not sets existed.
Updating a set fixes that set for a while, presumably until the next kodi restart.

Other users on my server don't seem affected, but I'm not completely sure on that.

**To Reproduce**
Create sets, I guess. Not sure why this happens

**Expected behavior**
For sets to work

**System (please complete the following information):**
 - OS: Android 11 (shield).
 - Jellyfin Version: latest linuxserver image. 10.7.7
 - Kodi Version: 19.3
 - Addon Version: 0.7.7+py3
 - Playback Mode: Native

**Additional context**
This used to work, not sure when it broke because I don't use movies that much but it's been a while.
One of my users told me they had no issues, but I don't know if they actually checked or understood what I was talking about so take that with a pinch of salt.

**Comment by mcarlton00** [comment]
Are you creating sets locally in Kodi, or syncing collections from the server?  I've never had any issues syncing collections from the server, but I've never tried to manually create sets in Kodi locally with Jellyfin content.

**Comment by Ulrar** [comment]
From the server, didn't realize you could do it from Kodi itself.
I've re-updated them again a few days ago and it seems to be holding, so whatever the issue is maybe it's been resolved ? I'll keep an eye on it !

**Comment by mcarlton00** [comment]
I would say it's probably not recommended to manage sets in Kodi alone.  They won't be synced back to the server afaik.

**Comment by Ulrar** [comment]
Right. I'm doing it on the server anyway, with that annoying collections folder on the home page to prove it

**Comment by krixon** [comment]
I'm seeing the same thing. Here are the steps I took:

- Sync everything from Jellyfin (no sets defined on the server yet).
- Install the box sets plugin on the server.
- Refresh metadata, movies now show in collections on the server.
- Refresh the library via the Kodi addon

At this point, no collections are created in the kodi DB (`SELECT COUNT(*) FROM sets;` -> `0`). If I look at the backup of my movie DB before switching to JF, I see this table populated with lots of rows.

Things I have tried:

- Refresh the library via the Kodi addon (`Manage Libraries -> Update libraries -> All`)
- Refresh the box set info via the Kodi addon (`Manage Libraries -> Refresh boxsets)

Both of these actions seem to complete without errors.

I'm not sure what to try next. Happy to provide any diagnostic info required.
